resources/definitions/fdmprinter.def.json

```diff
@@ -1230,6 +1230,16 @@
                     "default_value": true,
                     "enabled": "infill_sparse_density > 0",
                     "settable_per_mesh": true
+                },
+                "min_infill_area":
+                {
+                    "label": "Min Infill Area",
```

Comment:
We'll want this label to say "Minimum Infill Area" in full, rather than abbreviating it.